pushy-donkeys/workforce-management
# Km dept1

Changes proposed in this pull request:
- Added linq statement to DepartmentsController
-Added Employee Name to Department Details
-

How to test:
-git fetch
-git checkout km-dept1
-Run
-go to Departments and click on Details. Ensure that Employees full names are showing.

@PushyDonkey/
